doc/main/reference/blocking_terminate.rst

```diff
@@ -60,16 +62,43 @@ Synopsis
 Member Functions
 ----------------
 
+.. cpp:function:: task_scheduler_handle()
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that does not contain any reference to a task scheduler
+
+-------------------------------------------------------
+
 .. cpp:function:: ~task_scheduler_handle()
 
     **Effects**: Destroys an instance of the ``task_scheduler_handle`` class.
     Releases a reference to the task scheduler and deactivates an instance of the ``task_scheduler_handle`` class.
 
 -------------------------------------------------------
 
+.. cpp:function:: task_scheduler_handle(task_scheduler_handle&& other) noexcept
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that references to the task scheduler referenced with ``other``.
```

Comment:
Does ``other`` references the task scheduler or otherwise? Or do both reference to one another?